### ISSUE TITLE
Missing alt attribute

### ISSUE THREAD

**Issue by linkstrifer** [Junicavi/Junicavi.github.io#5]
All images should have an `alt` attribute for accessibility, this will be the text presented to users with vision problems

https://github.com/Junicavi/Junicavi.github.io/blob/86b98b35b7d8b45f37cd3323068c713276e75280/FrontEnd-Training/index.html#L29

https://github.com/Junicavi/Junicavi.github.io/blob/86b98b35b7d8b45f37cd3323068c713276e75280/FrontEnd-Training/index.html#L30

https://github.com/Junicavi/Junicavi.github.io/blob/86b98b35b7d8b45f37cd3323068c713276e75280/FrontEnd-Training/index.html#L38

https://github.com/Junicavi/Junicavi.github.io/blob/86b98b35b7d8b45f37cd3323068c713276e75280/FrontEnd-Training/index.html#L43

https://github.com/Junicavi/Junicavi.github.io/blob/86b98b35b7d8b45f37cd3323068c713276e75280/FrontEnd-Training/index.html#L48

https://github.com/Junicavi/Junicavi.github.io/blob/86b98b35b7d8b45f37cd3323068c713276e75280/FrontEnd-Training/index.html#L53

https://github.com/Junicavi/Junicavi.github.io/blob/86b98b35b7d8b45f37cd3323068c713276e75280/FrontEnd-Training/index.html#L58

https://github.com/Junicavi/Junicavi.github.io/blob/86b98b35b7d8b45f37cd3323068c713276e75280/FrontEnd-Training/index.html#L63

https://github.com/Junicavi/Junicavi.github.io/blob/86b98b35b7d8b45f37cd3323068c713276e75280/FrontEnd-Training/index.html#L94

https://github.com/Junicavi/Junicavi.github.io/blob/86b98b35b7d8b45f37cd3323068c713276e75280/FrontEnd-Training/index.html#L99

https://github.com/Junicavi/Junicavi.github.io/blob/86b98b35b7d8b45f37cd3323068c713276e75280/FrontEnd-Training/index.html#L104

https://github.com/Junicavi/Junicavi.github.io/blob/86b98b35b7d8b45f37cd3323068c713276e75280/FrontEnd-Training/index.html#L110

https://github.com/Junicavi/Junicavi.github.io/blob/86b98b35b7d8b45f37cd3323068c713276e75280/FrontEnd-Training/index.html#L114

https://github.com/Junicavi/Junicavi.github.io/blob/86b98b35b7d8b45f37cd3323068c713276e75280/FrontEnd-Training/index.html#L119

https://github.com/Junicavi/Junicavi.github.io/blob/86b98b35b7d8b45f37cd3323068c713276e75280/FrontEnd-Training/index.html#L133
